docs/content/tutorials/absence-of-change/_index.md

```diff
@@ -0,0 +1,187 @@
+---
+type: "docs"
+title: "Absence of Change"
+linkTitle: "Absence of Change"
+weight: 50
+description: >
+    Responding in the absence of changes
+---
+
+## Scenario
+
+In this tutorial, we will explore the use case where the requirement is to monitor a fleet of freezers and trigger an alert if one of them remains above 32 degrees continuously for more than 15 minutes, an alert must not fire if the freezer rises above 32 but drops back below that threshold in less than 15 minutes. There is no explicit event or mechanism that confirms the fact that the temperature has been above the threshold continuously for 15 minutes. For demonstration purposes, we will use a duration of 10 seconds instead of 15 minutes. The source data is a PostgreSQL table that stores the latest reported temperature of each freezer.
+
+
+### Tutorial Modes
+
+You can follow along the steps below in a Github codespace, a VSCode Dev Container or your own Kubernetes environment.
+
+{{< tabpane >}}
+
+{{% tab header="Github Codespaces" text=true %}}
+
+The easiest way to follow along with this tutorial is to launch a Github
+  Codespace using the link below. This will allow you to run the example
+  application within your browser without setting up anything on your own
+  machines.
+
+[![Open in Github Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/drasi-project/learning?devcontainer_path=.devcontainer%2Fabsence-of-change%2Fdevcontainer.json)
+
+This will open a page with some configuration options. Make sure that the
+  **Branch** selected is **main** and set the **Dev Container configuration** to **Absence of Change with Drasi**.
+
+{{% /tab %}}
+
+{{% tab header="VS Code Container" text=true %}}
+
+To follow along with a Dev Container, you will need to install:
+- [Visual Studio Code](https://code.visualstudio.com/)
+- Visual Studio Code [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) 
+- [docker](https://www.docker.com/get-started/)
+
+Next, [clone the learning repo from Github](https://github.com/drasi-project/learning),
+  and open the repo in VS Code. Make sure that Docker daemon
+  (or Docker Desktop) is running.
+
+Once the solution is open in VS Code, follow these steps:
+- Press Cmd + Shift + P (on MacOS) or Ctrl + Shift + P (Windows or Linux) to
+    launch the command palette.
+- Select `Dev Containers: Rebuild and Reopen in Container`.
+- Select the `Absence of Change with Drasi` option to launch this tutorial.
+
+{{% /tab %}}
+
+{{% tab header="Local Setup" text=true %}}
```

Comment:
I think we should also add the CLI as one of the prereqs for the local setup, as the CLI is later used for applying the files.